src/components/Modals/NewCourse/SelectedRequirementEditor.vue

```diff
@@ -1,75 +1,52 @@
 <template>
   <div>
-    <div v-if="chosenRequirementText.length > 0">
+    <div v-if="requirementsThatAllowDoubleCounting.length > 0">
       <div class="newCourse-title">
-        You will let this class fulfill the following requirements(s):
+        This class automatically fulfills the following requirement(s):
       </div>
-      <div class="newCourse-requirements-container">
-        <div class="newCourse-requirements">
-          {{ chosenRequirementText }}
-        </div>
-      </div>
-    </div>
-    <div v-if="requirementsThatAllowDoubleCounting.length > 0">
-      <div class="newCourse-title">This class can auto-fulfill the following requirement(s):</div>
       <div class="newCourse-requirements-container">
         <div class="newCourse-requirements">
           {{ requirementsThatAllowDoubleCounting.join(', ') }}
         </div>
       </div>
     </div>
-    <div v-if="relatedRequirements.length > 0">
+    <div v-else class="newCourse-requirements-container newCourse-requirements">
+      This class does not automatically fulfill any requirements
```

Comment:
Is this text supposed to be blue? Also, should there be a period?

Reply:
i realized @einc doesnt use periods lol. i agree that there should be a period. Ein, is it cool if i add a period or is this a design decision?

Reply:
no lol that's just my bad typing habits. yes haha pls add periods or capitalize stuff as normal <3